Easy Build Release.bat

```diff
@@ -0,0 +1,10 @@
+git status
+git fetch
```

Comment:
This does not do anything on its own except fetch metadata. These git operations can be done by the programmer.